README.md

```diff
@@ -87,10 +87,16 @@ Fork and clone this repository. Issue pull-requests one document at a time.
  ```
  If you use --watch and see it constantly running, regenerating, drop --watch
  until you identify and resolve the problem.  
-5. Images  
- For images to render on your local environment you need to apply these commands from within the `documentation` directory:
+
+4. Images:
+
+For images to render on your local environment you need to apply these commands from within the `documentation` directory:
 
  ```
  $ cd output_dev
  $ ln -s ./ source
  ```
+
+5. Compiling CSS changes:
```

Comment:
Ah, yeah. Line 93 needs a space added at the start of the line to follow markdown syntax, otherwise the ordered list is closed. I can clean the styling up if you'd like @stevector